src/vsock/src/bin/vhost_user_vsock.rs

```diff
@@ -0,0 +1,39 @@
+#[macro_use(crate_version, crate_authors)]
+extern crate clap;
+extern crate vhost_user_vsock;
+
+use clap::{App, Arg};
+use vhost_user_vsock::start_backend_server;
+use vhost_user_vsock::VsockConfig;
+
+fn main() {
```

Comment:
I am not sure what's the right way, but I will rather name this file as main.rs and move it to src/ itself.